packages/api/stryker.conf.js

```diff
@@ -1,7 +0,0 @@
-const path = require('path');
-const settings = require('../../stryker.parent.conf');
-const moduleName = __dirname.split(path.sep).pop();
-settings.dashboard.module = moduleName;
-module.exports = function (config) {
-  config.set(settings);
-};
```

Comment:
🤔

Reply:
Geen idee hoe dit is gebeurd, heb het ongedaan gemaakt.